rust/processor/migrations/2023-12-07-233224_add_objects_model/up.sql

```diff
@@ -0,0 +1,5 @@
+-- Your SQL goes here
+ALTER TABLE objects ADD COLUMN IF NOT EXISTS is_token BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE objects ADD COLUMN IF NOT EXISTS is_fungible_asset BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE current_objects ADD COLUMN IF NOT EXISTS is_token BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE current_objects ADD COLUMN IF NOT EXISTS is_fungible_asset BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
I don't think that we should default these to false. Isn't it wrong? IMO these should be nullable field that if unfilled should just be null.

Reply:
an object is either a token or it's not. i'll change it to nullable but after backfill these new columns should be filled for all rows